### PR TITLE
Correct patch content for plugin test step in pr-test.yml

### PR DESCRIPTION
**Pull request by JohnNiang**
This pr is trying to fix wrong patch json while testing plugin in pr-test.yml.

Let's have a look at incorrect patch raw json:

https://github.com/kubesphere/ks-installer/blob/dd5274f536bdcb52fa3738dfb196b9169b7a3d72/.github/workflows/pr-test.yml#L39

and incorrect patch formatted json:

```json
{
  "spec": {
    "alerting": { "enabled": true },
    "auditing": { "enabled": true },
    "enableMultiLogin": { "enabled": true },
    "events": { "enabled": true },
    "logging": { "enabled": true },
    "metrics_server": { "enabled": true },
    "network": { "networkpolicy": { "enabled": true } },
    "store": { "enabled": true }
  }
}
```

I've already convert the json format into yaml format:

```yaml
spec:
  alerting:
    enabled: true
  auditing:
    enabled: true
  events:
    enabled: true
  logging:
    enabled: true
  metrics_server:
    enabled: true
  network:
    networkpolicy:
      enabled: true
  openpitrix:
    store:
      enabled: true
```

Related to #1454

BTW, this PR is no hurry to merge.

/kind bug

### CODE REVIEW

**Comment by zackzhangkai** [comment]
Thanks for your PR @JohnNiang . But I think there is no need to add more files on `.github/workflows` directory. It will be seen too much messy. Patch raw json is a frequency action.

**Comment by JohnNiang** [comment]
> Thanks for your PR @JohnNiang . But I think there is no need to add more files on `.github/workflows` directory. It will be seen too much messy. Patch raw json is a frequency action.

Alright, the json patch is not quite right, you can have a check.

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
 
     - name: Test plugin
       run: |
-        kubectl -n kubesphere-system patch cc ks-installer --type merge --patch '{"spec":{"alerting":{"enabled":true},"auditing":{"enabled":true},"enableMultiLogin":{"enabled":true},"events":{"enabled":true},"logging":{"enabled":true},"metrics_server":{"enabled":true},"network":{"networkpolicy":{"enabled":true}},"store":{"enabled":true}}}'
```
Comment (zackzhangkai) [inline .github/workflows/pr-test.yml:39]: why not remove enableMultiLogin?

```bash
  console:
    enableMultiLogin: true
    port: 30880
```

Comment (JohnNiang) [reply .github/workflows/pr-test.yml:39]: Thanks for your reminder.

**Comment by zackzhangkai** [comment]
@JohnNiang  Please rebase your commit.

**Comment by zackzhangkai** [comment]
Please sign off your pr.  You can do it  appand '-s' , such as `git commit -m "xxx" -s`

**Comment by JohnNiang** [comment]
@zackzhangkai Thanks for your review. It's recommended that this PR should be squashed into master instead of merging directly, for some unnecessary commits here. Please let me know your suggestion : P

**Comment by zackzhangkai** [comment]
Just rebase is ok. Such as: 

```bash
git fetch upstream/master
git rebase -i upstream/master

pick 56ac7bcd fix send notification
f 05f4aaed fix send notification  # change this to f
```

**Comment by zackzhangkai** [comment]
@pixiake For the first-time contributor, please run workflows by hand. 

https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Pull request merged**